x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result_header.scss

```
@@ -0,0 +1,26 @@
+.appSearchResultHeader {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+
+  &Column {
```

I have a very slight to medium preference to not nesting this under `.appSearchResultHeader` (since it's not technically related by BEM) and instead pulling it out to its own block:

```suggestion
}

.appSearchResultHeaderColumn {
```

Doing so also makes it much easier to grep the codebase for the full CSS class name (when searching from the DOM markup) and quickly find the corresponding styles

---

I strongly agree.

---

b9e1126c042d6249514696588992721111cd1689